diff --git a/lib/get-token.js b/lib/get-token.js
index a0d04e4ec5370924e512e759724c67eae62b2baf..c061cd59cc61cc6403a14cb1b410e3d5f5f11f9c 100644
--- a/lib/get-token.js
+++ b/lib/get-token.js
@@ -9,7 +9,7 @@ import {
   base64encode,
 } from "./utils.js";
 
-import { subtle, convertPrivateKey } from "#crypto";
+import { subtle, convertPrivateKey } from "./crypto-node.js";
 
 /**
  * @param {import('../internals').GetTokenOptions} options
